concepticondata/conceptlists/Smith-2017-680.tsv

```diff
@@ -0,0 +1,681 @@
+297	index finger	577	FOREFINGER
+298	in front	1883	IN FRONT OF
+299	insect bat
+300	inside	1606	INSIDE
+301	insult	3204	INSULT
+302	intestinal worms	3623	INTESTINAL WORM
+303	intestines	1991	INTESTINES
+304	ironwood
+305	island	622	ISLAND
+306	itchy	149	ITCHY
+307	jealous	2821	JEALOUS
+308	jump	1829	JUMP
+309	kick	1135	KICK
+310	kill	1417	KILL
+311	kiss	578	KISS
+312	knee	1371	KNEE
+313	knife	1352	KNIFE
+314	knuckle	2824	KNUCKLE
+315	lake	624	LAKE
+316	land	626	LAND
+317	large fly
+318	large mortar
+319	large pestle
+320	laugh	1355	LAUGH
+321	lazy	1564	LAZY
+322	leaf	628	LEAF
+323	leak	2253	DRIP (EMIT LIQUID)
+324	learn	504	LEARN
+325	leave	1752	LEAVE
+326	leech	2273	LEECH
+327	left	244	LEFT
+328	leg	1297	LEG
+329	lend	582	LEND
+330	leopard	1139	LEOPARD
+331	lid	2319	LID (COVER, CAP)
+332	lie down	215	LIE DOWN
+333	lift, carry
+334	light/bright
+335	lightening	1654	LIGHTNING
+336	light weight	1052	LIGHT (WEIGHT)
+337	like	1117	LIKE
+338	lime	982	LIME (FRUIT)
+339	lips	478	LIP
+340	listen, hear	3862	HEAR OR LISTEN
+341	liver	1224	LIVER
+342	lizard	632	LIZARD
+343	loincloth	1965	LOINCLOTH
+344	long	1203	LONG
+345	loose	2506	LOOSE
+346	lost	3667	LOST
+347	loud	377	LOUD
+348	louse	1392	LOUSE
+349	lowland plain	652	PLAIN
+350	make	734	MAKE
+351	man	1554	MAN
+352	mantis	3279	MANTIS
+353	many	1198	MANY
+354	married
+355	mat	195	MAT
+356	measure	1506	MEASURE
+357	meet	1831	MEET
+358	millipede
+359	mist	249	FOG
+360	mix	2017	MIX
+361	molars
+362	monitor
+363	monkey	1350	MONKEY
+364	moon	1313	MOON
+365	morning	1339	MORNING
+366	mosquito	1509	MOSQUITO
+367	moss	637	MOSS
+368	mother	1216	MOTHER
+369	mother in law	1050	MOTHER-IN-LAW
+370	mountain	639	MOUNTAIN
+371	mouse deer
+372	mouth	674	MOUTH
+373	mud	640	MUD
+374	mushroom	641	MUSHROOM
+375	mute	1666	MUTE
+376	naked	590	NAKED
+377	name	1405	NAME
+378	nape	1347	NAPE (OF NECK)
+379	narrow	1267	NARROW
+380	navel	1838	NAVEL
+381	near	1942	NEAR
+382	neck	1333	NECK
+383	nest	539	NEST
+384	new	1231	NEW
+385	next to	2917	NEXT TO
+386	niece; nephew	3890	NEPHEW OR NIECE
+387	night	1233	NIGHT
+388	nine	1483	NINE
+389	nipple	796	NIPPLE
+390	noon	12	MIDDAY
+391	nose	1221	NOSE
+392	now	1376	NOW
+393	ocean	645	OCEAN
+394	old	1229	OLD
+395	older siblings	405	OLDER SIBLING
+396	one	1493	ONE
+397	only	1580	ONLY
+398	on top	57	ATOP
+399	open	1156	OPEN
+400	or	1014	OR
+401	otter	15	OTTER
+402	overflow	3130	SPILL (SOMETHING)
+403	Owl	735	OWL
+404	paddle	234	PADDLE
+405	painful	1129	PAINFUL
+406	palate	540	PALATE
+407	palm	1183	PALM OF HAND
+408	pangolin	2508	PANGOLIN
+409	parang	2680	MACHETE
+410	peak/top	2205	PEAK
+411	peel fruit
+412	person	683	PERSON
+413	pinch	378	PINCH
+414	pinky	2774	LITTLE FINGER
+415	pit viper
+416	place, noun	723	PLACE (POSITION)
+417	plant	653	PLANT (VEGETATION)
+418	plate	480	PLATE
+419	plateau
+420	pluck	2148	PICK
+421	point	2669	POINT (WITH FINGER)
+422	pond	2035	POND
+423	porcupine	2510	PORCUPINE
+424	pound rice
+425	pour	1487	POUR
+426	pregnant	1123	PREGNANT
+427	primary jungle
+428	print
```

Comment:
Potentially a candidate for a new concept (however, a potentially rather ambigous one).

Reply:
428 is in reference to virgin rain forest. Or that which has not been touched by human hands. Typically land that has never been converted for farming. 

429 is an animal print or foot print.

Reply:
I wouldn't add it as a concept because this could lead to many ambiguous mappings.

Reply:
I'm not adding this in the list of potential new concepts in the issue, then :)